From 7d554d0c786f5431ad3035a328815890a95a2979 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=89rico=20Nogueira?= <ericonr@disroot.org>
Date: Wed, 12 May 2021 17:39:20 -0300
Subject: [PATCH] remove O_LARGEFILE

useful hack to build 1.2.2 musl for x86_64 and not have to rebuild
firefox.
---
 arch/generic/bits/fcntl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/generic/bits/fcntl.h b/arch/generic/bits/fcntl.h
index 730a98cf..c2fd1760 100644
--- a/arch/generic/bits/fcntl.h
+++ b/arch/generic/bits/fcntl.h
@@ -13,7 +13,7 @@
 
 #define O_ASYNC      020000
 #define O_DIRECT     040000
-#define O_LARGEFILE 0100000
+#define O_LARGEFILE 0
 #define O_NOATIME  01000000
 #define O_PATH    010000000
 #define O_TMPFILE 020200000
-- 
2.31.1
